.github/workflows/test-mongodb.yml

```diff
@@ -12,17 +12,11 @@ name: "Tests Mongodb"
 jobs:
     Tests:
         runs-on: ubuntu-latest
-        services:
-            mongo:
-                image: 'mongo:latest'
-                ports:
-                    - '27017:27017'
         strategy:
             matrix:
                 go-version:
-                    - 1.19.x
-                    - 1.20.x
                     - 1.21.x
+                    - 1.22.x
```

Comment:
Once tests are green, re-add 1.23 here

Reply:
So we want to include 1.23 in these new tests? The matrix would be 1.21, 1.22 and 1.23, right?

Reply:
Correct, 1.23 should work